lib/SILOptimizer/Mandatory/TFConstExpr.cpp

```diff
@@ -712,6 +757,33 @@ ConstExprFunctionState::computeConstantValueBuiltin(BuiltinInst *inst) {
   return unknownResult();
 }
 
+// Handle calls to opaque callees, either by handling them and returning None or
+// by returning with a Unknown indicating a failure.
+llvm::Optional<SymbolicValue>
+ConstExprFunctionState::computeOpaqueCallResult(ApplyInst *apply,
+                                                SILFunction *callee) {
+  // This is a termination point like fatalError.
+  if (callee->hasSemanticsAttr("arc.programtermination_point")) {
+    // TODO: Actually get the message out of fatalError.  We can literally get
+    // its string and file/line/col info and propagate it up!
+    return SymbolicValue::getUnknown((SILInstruction*)apply,
+                                     UnknownReason::Trap);
+  }
+
+  if (classifyFunction(callee->getName()) ==
```

Comment:
Pure nit: Use `{}` here also for consistency since the last if-statement did?

Reply:
Sure